app/controllers/users_controller.rb

```diff
@@ -1,10 +1,18 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, except: %i(new show create)
+  before_action :correct_user, only: %i(edit update)
+  before_action :verify_admin, only: :destroy
+  before_action :load_user, except: %i(index new create)
+
+  def index
+    @users = User.page(params[:page]).per Settings.user_controller.per_paginate
+  end
+
   def new
     @user = User.new
   end
 
   def show
-    @user = User.find_by id: params[:id]
     return if @user
     flash[:warning] = t :show
     redirect_to root_path
```

Comment:
remove the body of the show method as you are doing the same while loading user in the before_action filter